docs/upgrade/automatic.md

```diff
@@ -34,7 +34,7 @@ Note we are still working towards zero-downtime upgrade, due to some known issue
     - Back up your VMs if needed.
 - Do not operate the cluster during an upgrade. For example, creating new VMs, uploading new images, etc.
 - Make sure your hardware meets the **preferred** [hardware requirements](../index.md#hardware-requirements). This is due to there will be intermediate resources consumed by an upgrade.
-- Make sure each node has at least 25 GB of free space (`df -h /usr/local/`).
+- Make sure each node has at least 30 GiB of free space (`df -h /usr/local/`). If any node in the cluster has fewer free space, the upgrade will be denied. Check [free space requirement](#free-space-requirement) for more information.
```

Comment:
```suggestion
- Make sure each node has at least 30 GiB of free space (`df -h /usr/local/`). If any node in the cluster has less than 30 GiB of free space, the upgrade will be unsuccessful. Check [free space requirement](#free-space-requirement) for more information.
```

Reply:
@LucasSaintarbor Here a window is poped immediately to say upgrade is denied. It is more accurate than `unsuccessful`.